core/src/main/resources/jenkins/model/Jenkins/legend_ko.properties

```
@@ -20,21 +20,20 @@
-health-81plus=\uD504\uB85C\uC81D\uD2B8 \uAC74\uAC15\uC0C1\uD0DC\uAC00 28% \uCD08\uACFC\uC784. \uB9C8\uC6B0\uC2A4\uB97C \uD504\uB85C\uC81D\uD2B8 \uC544\uC774\uCF58\uC5D0 \uC62C\uB824\uB193\uC73C\uBA74 \uB354 \uC790\uC138\uD55C \uC124\uBA85\uC744 \uBCFC \uC218 \uC788\uC2B5\uB2C8\uB2E4.
-red=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uC2EC\uAC01\uD558\uAC8C \uC2E4\uD328\uD568.
-red_anime=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uC2EC\uAC01\uD558\uAC8C \uC2E4\uD328\uD568. \uC2E0\uADDC \uBE4C\uB4DC\uAC00 \uC9C4\uD589 \uC911\uC784.
-yellow=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uC131\uACF5\uD588\uC9C0\uB9CC \uBD88\uC548\uC815\uD568. \uC8FC\uB85C \uB0A8\uC544\uC788\uB358 \uAC80\uC99D \uC2E4\uD328\uB97C \uC0AC\uC6A9\uD55C \uAC83\uC784.
-yellow_anime=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uC131\uACF5\uD588\uC9C0\uB9CC \uBD88\uC548\uC815\uD568. \uC2E0\uADDC \uBE4C\uB4DC\uAC00 \uC9C4\uD589 \uC911\uC784
+lightgrey=\uD504\uB85C\uC81D\uD2B8\uAC00 \uBE4C\uB4DC\uB41C \uC801\uC774 \uC5C6\uC2B5\uB2C8\uB2E4.
+lightgrey_anime=\uD504\uB85C\uC81D\uD2B8\uC758 \uCD5C\uCD08 \uBE4C\uB4DC\uAC00 \uC9C4\uD589 \uC911\uC785\uB2C8\uB2E4.
+grey=\uD504\uB85C\uC81D\uD2B8\uAC00 \uBE4C\uB4DC\uB41C \uC801\uC774 \uC5C6\uAC70\uB098, \uBE44\uD65C\uC131 \uC0C1\uD0DC\uC785\uB2C8\uB2E4.
+grey_anime=\uD504\uB85C\uC81D\uD2B8\uC758 \uCD5C\uCD08 \uBE4C\uB4DC\uAC00 \uC9C4\uD589 \uC911\uC785\uB2C8\uB2E4.
+darkgrey=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uCDE8\uC18C\uB418\uC5C8\uC2B5\uB2C8\uB2E4.
+darkgrey_anime=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uCDE8\uC18C\uB418\uC5C8\uC2B5\uB2C8\uB2E4. \uC0C8\uB85C\uC6B4 \uBE4C\uB4DC\uAC00 \uC9C4\uD589 \uC911\uC785\uB2C8\uB2E4.
+blue=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uC131\uACF5\uD588\uC2B5\uB2C8\uB2E4.
+blue_anime=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uC131\uACF5\uD588\uC2B5\uB2C8\uB2E4. \uC0C8\uB85C\uC6B4 \uBE4C\uB4DC\uAC00 \uC9C4\uD589 \uC911\uC785\uB2C8\uB2E4.
+yellow=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uC131\uACF5\uD588\uC9C0\uB9CC \uBD88\uC548\uC815\uD569\uB2C8\uB2E4. \uC774\uAC83\uC740 \uC8FC\uB85C \uD14C\uC2A4\uD2B8 \uC2E4\uD328\uB97C \uB098\uD0C0\uB0B4\uB294 \uB370 \uC0AC\uC6A9\uB429\uB2C8\uB2E4.
+yellow_anime=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uC131\uACF5\uD588\uC9C0\uB9CC \uBD88\uC548\uC815\uD569\uB2C8\uB2E4. \uC0C8\uB85C\uC6B4 \uBE4C\uB4DC\uAC00 \uC9C4\uD589 \uC911\uC785\uB2C8\uB2E4.
+red=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uC2E4\uD328\uD588\uC2B5\uB2C8\uB2E4.
+red_anime=\uB9C8\uC9C0\uB9C9 \uBE4C\uB4DC\uAC00 \uC2E4\uD328\uD588\uC2B5\uB2C8\uB2E4. \uC0C8\uB85C\uC6B4 \uBE4C\uB4DC\uAC00 \uC9C4\uD589 \uC911\uC785\uB2C8\uB2E4.
+health-81plus=\uD504\uB85C\uC81D\uD2B8 \uAC74\uAC15\uC0C1\uD0DC\uAC00 81% \uCD08\uACFC\uC785\uB2C8\uB2E4. \uB9C8\uC6B0\uC2A4\uB97C \uD504\uB85C\uC81D\uD2B8 \uC544\uC774\uCF58\uC5D0 \uC62C\uB824\uB193\uC73C\uBA74 \uB354 \uC790\uC138\uD55C \uC124\uBA85\uC744 \uBCFC \uC218 \uC788\uC2B5\uB2C8\uB2E4.
```

For consistency, this should probably say
```suggestion
health-81plus=\uD504\uB85C\uC81D\uD2B8 \uAC74\uAC15\uC0C1\uD0DC\uAC00 80% \uCD08\uACFC\uC785\uB2C8\uB2E4. \uB9C8\uC6B0\uC2A4\uB97C \uD504\uB85C\uC81D\uD2B8 \uC544\uC774\uCF58\uC5D0 \uC62C\uB824\uB193\uC73C\uBA74 \uB354 \uC790\uC138\uD55C \uC124\uBA85\uC744 \uBCFC \uC218 \uC788\uC2B5\uB2C8\uB2E4.
```
At least the other entries use the round number. I don't know any Korean so that may well be wrong then (and need adjustments elsewhere).

---

You are right. The original aim to fix is the number, it was 28% and now 81% which is also wrong :( I will fix it to 80%.